### PR TITLE
estabilish postgres connection; save / show data

### PR DESCRIPTION
**Pull request by chaserx**

### CODE REVIEW

**Comment by chaserx** [comment]
updated PR title. we can save and retrieve data from PG DB.

File: server.go [hunk 1]
```diff
@@ -59,6 +73,59 @@ func hacker_handler(w http.ResponseWriter, r *http.Request) {
...
+  // return JSON
+  w.Header().Set("Content-Type", "application/json")
+  w.Write(js)
+}
+
+func today(h string) bool {
   return false
 }
+
+func InitEnv() {
```
Comment (millidavids) [inline server.go:106]: I don't understand what this does. Is it just checking to see if it can access the environment?

Comment (chaserx) [inline server.go:106]: loads up the `.env` file with godotenv.

Comment (chaserx) [inline server.go:106]: kinda like the dotenv gem in ruby land

**Comment by nikolaiwarner** [comment]
+1

**Pull request merged**